test-infra/image-builder/roles/kubevirt-images/tasks/main.yml

```diff
@@ -27,6 +35,8 @@
     - "{{ images }}"
   when:
     - item.value.converted|bool
+    # .qcow2.xz images have been uncompressed as .qcow2
+    - item.value.filename.find('.qcow2.xz') == -1
```

Comment:
Maybe `not endswith` would be easier to follow? But why not setting `converted: false` in the first place?

My understanding is that this works because `filename` is `{{ item.key }}.qcow2`?

Reply:
`filename` is `{{ item.key }}.qcow2.xz` so it's uncompressed as `{{ item.key }}.qcow2`. So the cp command will copy the file to itself:
```bash
cp {{ images_dir }}/fedora-coreos.qcow2 {{ images_dir }}/fedora-coreos.qcow2
```
That's why I skip this step for `*.qcow2.xz` images. But if you prefer i can `unxz` the file with a different name and keep the `cp` for clarity?

Reply:
Nevermind, i changed it to keep the `cp` step.